spec/std/spec_spec.cr

```diff
@@ -117,6 +117,9 @@ describe "Spec matchers" do
 end
 
 describe "Spec" do
+  it "should treat it calls without blocks as pending"
+  pending "should allow pending calls without blocks"
```

Comment:
Both of these add two "pending" test cases to the output. I'm open to other ideas on how to test this. The compiler does a really good job of providing a "failing" test case without the new methods in `spec/methods.cr` though.

Reply:
Just don't test it. The code is trivially simple.

Reply:
👍